unabridged-works-volume-2/original/07-common-objection-answered.adoc

```diff
@@ -219,9 +219,9 @@ over which we cannot but mourn, and wait for its breaking off the chains,
 and its rising out of all its graves into its own pure life, power,
 and fulness of liberty in the Lord.
 
-=== A LOVING AND FAITHFUL ADVERTISEMENT TO THE NATION, AND POWERS THEREOF
+=== A Loving and Faithful Advertisement to the Nation, and Powers Thereof
```

Comment:
I wasn't sure if I should embed this because it has a signature and a part that is a postscript.

Reply:
Yeah, I see why you wondered that.  I think it's good to leave it not embedded, because it's not embedding another document, it's really just the way he wrote this particular treatise. So, you did the right thing.